<?php

namespace _PhpScoper006a73f0e455\Bug3683;

function (\Generator $g) : void {
    $g->throw(new \Exception());
    $g->throw(1);
};
